contracts/test/helpers/ACLHelper.sol

```diff
@@ -57,3 +57,10 @@ contract ConditionalOracle is IACLOracle {
         return how[0] > 0;
     }
 }
+
+contract OverGasLimitOracle is IACLOracle {
+    function canPerform(address, address, bytes32, uint256[]) external view returns (bool) {
+        while (true) {}
+        return true;
```

Comment:
As mentioned in the call, we could set a random storage slot to a non-zero value here to make our tests a bit faster :).

Reply:
Used a memory slot instead as it's a view function, to prevent confusion. Test still much faster.